### PR TITLE
fix: i18n problems with content tracker title etc

### PR DESCRIPTION
**Pull request by magicus**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java [hunk 1]
```diff
@@ -78,10 +78,12 @@ protected ContentTrackerOverlay() {
 
             toRenderPreview
                     .get(0)
-                    .setText(I18n.get("feature.wynntils.contentTrackerOverlay.overlay.tracker.title") + " Quest:");
+                    .setText(I18n.get("feature.wynntils.contentTrackerOverlay.overlay.contentTracker.title")
+                            + " Quest:");
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java:82]: Technically Quest could be an i18n..

**Review by kristofbolyai — APPROVED** [review]

File: common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java:82]: I think we have hard coded texts for preview elsewhere.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java:82]: Oh previews.. nevermind then

Comment (magicus) [reply common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java:82]: Like two lines below :-D.

**Pull request merged**